technical/indicators.py

```diff
@@ -274,6 +278,8 @@ def ichimoku(dataframe, conversion_line_period=9, base_line_periods=26,
         'kijun_sen': kijun_sen,
         'senkou_span_a': senkou_span_a,
         'senkou_span_b': senkou_span_b,
+        'future_senkou_span_a': future_senkou_span_a,
+        'future_senkou_span_b': future_senkou_span_b,
```

Comment:
I would prefer to call them "leading_senkou_*" - that's how [investopedia](https://www.investopedia.com/terms/i/ichimoku-cloud.asp) calls them too.

Reply:
thanks for the great suggestions, implemented them in the last commit